nuget.config

```diff
@@ -9,8 +9,5 @@
        </repository>
      </trustedSigners>
    </config>
-   <packageSources>
-        <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
```

Comment:
Why did you remove this line?

Reply:
Because it was in mu list of package sources twice. Is it needed for your setup?

Reply:
OK. I don't think so (not checked :-)) .